index.html

```diff
@@ -216,6 +212,38 @@ <h6 class="about-title">About Myself</h6>
           <!--End of Frameworks-->
         </div>
       </section> 
+      <!-- the contact section -->
+      <footer class="contact-section">
+        <div class="title-content-container">
+          <p class="title">Contact Me</p>
+          <p class="content">
+            If you have an application you are 
+            interested in developing, a feature that you need built 
+            or a project that needs coding. I’d love to help with it 
+          </p>
+        </div>
+        <div class="contact-form-container">
+          <form action="https://formspree.io/f/xjvlljqr" method="post">
+            <div class="form-group">
+              <input type="text" maxlength="30" placeholder="Osika Destee" required>
+              <div title="required"></div>
+            </div>
+            <div class="form-group">
+              <input id="email" type="email" placeholder="Osikadestee360@gmail.com" required>
+              <div title="required"></div>
+            </div>
+            <div class="form-group">
+              <textarea name="message" placeholder="Write your message here." id="message" cols="30" rows="10" required></textarea>
+              <div title="required"></div>
+            </div>
+            <input type="submit" class="submit" value="Get it touch">
```

Comment:
- [ ] Kindly make this submit input interactive with  some CSS pseudo classes like :hover
    
![Screenshot from 2022-05-31 15-38-21](https://user-images.githubusercontent.com/66097371/171201086-2b380470-33f4-45bd-96b1-1224fb9b64a5.png)